src/base/coordinate-grid-mixin.ts

```diff
@@ -1220,11 +1191,6 @@ export class CoordinateGridMixin extends ColorMixin(MarginMixin) {
         if (this._conf.zoomOutRestrict) {
             // ensure range is within self._xOriginalDomain
             range = this._checkExtents(range, this._xOriginalDomain);
-
-            // If it has an associated range chart ensure range is within domain of that rangeChart
-            if (this._rangeChart) {
```

Comment:
@gordonwoodhull this seems to be the only functional code that is getting dropped.

Reply:
Oh, I get it now, sorry I misunderstood.

I should not try to review PRs without trying them out!

Yes, I agree this is an obscure case. I misread your earlier comment "the earlier code was also checking the domain of the other chart and constraining based on that" and thought it meant that pan and zoom on the focus chart would not set the brush on the range chart, but of course it still does that.

Sorry again for the confusion.

Reply:
Thanks, @gordonwoodhull  for the review. :+1:

Reply:
English is my 3rd language and communication definitely not my strongest suit :smile:

Reply:
dc.js is sufficiently complex that it’s not easy to write about clearly.

A lot of my own comments mystify me only months later. 🤣